platforms/allwinner-d1/boards/lichee-rv/src/main.rs

```diff
@@ -74,6 +77,71 @@ fn main() -> ! {
         w
     });
 
+    // Initialize SPI stuff
+    k.initialize(async move {
+        SpiSender::register(k, 4).await.unwrap();
+    })
+    .unwrap();
+
+    k.initialize(async move {
+        k.sleep(Duration::from_millis(100)).await;
+        let mut spim = SpiSender::from_registry(k).await.unwrap();
```

Comment:
it would be nice if `SpiSender::register` could return a handle --- that way, we could await the `JoinHandle` of the first `k.initialize(...)` call in the second one, instead of having to sleep and then try to get it from the registry. Just a thought.